packages/jest-circus/src/unhandledRejectionHandler.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import type {Circus} from '@jest/types';
+import {addErrorToEachTestUnderDescribe, invariant} from './utils';
+
+/**
+ * Let's keep the original setTimeout for the usage below so we do not need to care about usage of fake timers in tests.
+ */
```

Comment:
I should probably rephrase this to follow similar pattern elsewhere in the circus code.

Reply:
consistency seems good 😀

Reply:
Done